daml-assistant/daml-helper/src/DA/Daml/Helper/Util.hs

```diff
@@ -197,3 +202,22 @@ waitForHttpServer sleep url headers = do
     where isIOException e = isJust (fromException e :: Maybe IOException)
           isHttpException e = isJust (fromException e :: Maybe HTTP.HttpException)
 
+tokenFor :: [T.Text] -> T.Text -> T.Text -> T.Text
+tokenFor parties ledgerId applicationId =
+  JWT.encodeSigned
+    (JWT.HMACSecret "secret")
+    mempty
+    mempty
+      { JWT.unregisteredClaims =
+          JWT.ClaimsMap $
+          Map.fromList
+            [ ( "https://daml.com/ledger-api"
+              , A.Object $
+                HashMap.fromList
+                  [ ("actAs", A.toJSON parties)
+                  , ("readAs", A.toJSON parties)
```

Comment:
The empty list is good enough here since everybody in `actAs` as `readAs` privileges as well.

Reply:
I'll remove it.